src/webaudio.js

```diff
@@ -135,9 +128,7 @@ export default class WebAudio extends util.Observer {
         this.peaks = null;
         /** @private */
         this.playbackRate = 1;
-        /** @private */
         this.analyser = null;
-        /** @private */
         this.scriptNode = null;
         /** @private */
         this.source = null;
```

Comment:
@marizuccara can you replace `/** @private */` with a meaningful comment for each variable?

Reply:
can you also add `@since` in new classes and/or methods? https://jsdoc.app/tags-since.html